selection-to-clipboard/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "manifest_version": 2,
+    "name": "selection-to-clipboard",
+    "description": "Example of WebExtensionAPI for writing to the [clipboard](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Interact_with_the_clipboard) ",
```

Comment:
Putting markdown in the description isn't supported by Firefox, it looks like this. Could you move that to readme please? 

<img width="767" alt="screenshot 2016-10-31 11 33 45" src="https://cloud.githubusercontent.com/assets/74699/19866608/06046a1e-9f5e-11e6-9589-5d9f2fa05136.png">